Makefile

```diff
@@ -112,6 +112,18 @@ pause-container-release: pause-container
 	mkdir -p "$(shell pwd)/out"
 	@docker save ${PAUSE_CONTAINER_IMAGE}:${PAUSE_CONTAINER_TAG} > "$(shell pwd)/out/${PAUSE_CONTAINER_TARBALL}"
 
+# Variable to determine branch/tag of amazon-ecs-cni-plugins
+revision=master
+cni-sources:
+	@git clone https://github.com/aws/amazon-ecs-cni-plugins.git --branch $(revision)
+
+cni-plugins: cni-sources
```

Comment:
> If we decouple these two targets we would end up with a fetch + build ideology.

I think that's ok because `release` and `docker` do the fetch part anyway. The only use-case where you want the freedom to do these yourself is during development and the assumption is that you would be pointing it to a different location than the standard `./amazon-ecs-cni-plugins` anyway.